.github/workflows/dashboards-observability-test-and-build-workflow.yml

```diff
@@ -51,11 +53,20 @@ jobs:
         with:
           path: OpenSearch-Dashboards/plugins/dashboards-observability
 
+      - name: Load Plugin Bootstrap cache
+        uses: actions/cache@v3
+        with:
+          path: |
+            **/node_modules
+            **/target
+          key: ${{ runner.os }}-bootstrap-${{ hashFiles('OpenSearch-Dashboards/yarn.lock', 'OpenSearch-Dashboards/plugins/dashboards-observability/yarn.lock') }}
+          restore-keys: ${{ runner.os }}-bootstrap-
 
       - name: Plugin Bootstrap
         run: |
-          cd OpenSearch-Dashboards/plugins/dashboards-observability
-          yarn osd bootstrap
+          cd OpenSearch-Dashboards
+          yarn config set network-timeout 1000000 -g
```

Comment:
Is this seconds?

Reply:
Milliseconds, this PR uses the same number as is used in the [OpenSearch Dashboards CI](https://github.com/opensearch-project/OpenSearch-Dashboards/commit/038968d154c8bcc9d9c75e7abbf9b9435b9a0e42), which in turn is inspired by a [related issue on yarn](https://github.com/yarnpkg/yarn/issues/8242#issuecomment-776561223) that suggests a similar approach with a number that's also in that ballpark.